### ISSUE TITLE
v2: firebase sometimes is not initialized during schedule load

### ISSUE THREAD

**Issue by echoAlexey** [gdg-x/hoverboard#379]
When user navigates to schedule or other page (or does multiple refresh) it is possible that Firebase will initialize after actions done to retrieve data, this causes errors and page doesn't load properly.

```
hoverboard-app.html:25454 Uncaught (in promise) ReferenceError: firebase is not defined
    at resolve (hoverboard-app.html:25454)
    at new Promise (<anonymous>)
    at Object.fetchUserFeaturedSessions (hoverboard-app.html:25453)
    at HTMLElement._fetchFeaturedSessions (schedule-page.html:250)
    at Object.runMethodEffect [as fn] (hoverboard-app.html:3152)
    at runEffectsForProperty (hoverboard-app.html:2498)
    at runEffects (hoverboard-app.html:2464)
    at HTMLElement._propertiesChanged (hoverboard-app.html:4012)
    at HTMLElement._flushProperties (hoverboard-app.html:1811)
    at HTMLElement.ready (hoverboard-app.html:3970)
```

25454 references to

```
 fetchUserFeaturedSessions: userId => {
    const result = new Promise(resolve => {
      firebase.database()
        .ref(`/featuredSessions/${userId}`)
        .on('value', snapshot => {
          resolve(snapshot.val());
        })
    });
```

**Comment by echoAlexey** [comment]
I was able to fix this by replacing firebase script init to 
  <script src="https://www.gstatic.com/firebasejs/4.9.1/firebase.js"></script> 

I can submit PR if you wish, it works for us :)

**Comment by ozasadnyy** [comment]
The reason we didn't do that way - it will block the rendering. 
For sure, your solution is working but we should find a more elegant solution. Does it make sense?

**Comment by echoAlexey** [comment]
Yeah, I understand.
Problem with firebase is quite often when someone is navigating via link to the page. When data get requested faster than firebase was loaded - no information will be shown, scripts throw exception with undefined.

We got multiple responses from people that schedule doesnt load, speakers do not display... from business perspective that's worse than "blocking rendering".

**Comment by ozasadnyy** [comment]
Should be fixed after #440 . Feel free to reopen if you will be able to reproduce the issue
